packages/account/src/Components/self-exclusion/__tests__/self-exclusion-inputs.spec.tsx

```diff
@@ -1,22 +1,69 @@
 import React from 'react';
-import { Formik } from 'formik';
+import { Formik, FormikContextType } from 'formik';
 import * as formik from 'formik';
 import { fireEvent, render, screen } from '@testing-library/react';
 import SelfExclusionInputs from '../self-exclusion-inputs';
-import SelfExclusionContext from '../self-exclusion-context';
+import SelfExclusionContext, { TSelfExclusionContext } from '../self-exclusion-context';
 
-jest.mock('../self-exclusion-footer', () => () => <div>SelfExclusionFooter</div>);
+jest.mock('../self-exclusion-footer', () => {
+    const MockSelfExclusionFooter = () => <div>SelfExclusionFooter</div>;
```

Comment:
So if i use the previous code it will give me warning ask me to give a variable name to the function. This is one of the way to solve the warning. Any idea on solving this issue? 🙏